docs/91_version_history.rst

```diff
@@ -33,175 +33,111 @@ Release Date: 2025-03-13
-       - OMF North plugin: sending OMF Data messages to the Edge Data Store 2020 resulted in the HTTP error code 400 with the message \_One or more errors occurred. (The action 'Update' is not supported for OMF messages.)_.\ The same problem occurs when sending OMF Data messages to the AVEVA Connector Relay. This has been fixed. This problem does not occur in EDS 2023, EDS 2023 Patch 1 and EDS 2024.
-       - 
-       - 
-       - 
-       - An issue that would occasionally cause the addition of the Observer south plugin to a disabled service to crash has been resolved.
+       - fledge-south-s2opcua: for the Asset Naming Scheme selection of Single Datapoint or Single Asset, Browse Names of Variables must be unique among all OPC UA Subscriptions. If there are duplicate Browse Names, the plugin should disambiguate the names by concatenating the Variable’s Node Id to the end of the Browse Name. If there are more than 2 duplicate Browse Names, the plugin would concatenate the Node Id too many times. This has been fixed.
+       - An issue with the fledge-south-mqtt-sparkplug plugin that would allow illegal asset names to be altered has been resolved.
+       - An issue with the dynamic reconfiguration of the fledge-south-randomwalk plugin has been resolved. The service no longer requires a restart after reconfiguration.
+       - An issue with the fledge-south-opcua that could cause problems if the service is restarted when the connection to the OPC UA server is not available has been resolved. 
+       - An issue that could cause the fledge-south-opcua plugin to fail when reconfigured has been resolved.
+       - A problem with the fledge-south-mqtt plugin that would cause it to not re-establish the connection to the MQTT broker if connectivity was lost has been resolved.
+       - An issue that could cause the rules in the fledge-filter-asset to be executed in the wrong order has been fixed.
+       - An issue that might result in the fledge-filter-asset plugin not removing all the datapoints from an asset, defined by multiple rules, has been resolved.
+       - A problem that would cause the fledge-filter-asset plugin to exit in a non-graceful way if an incomplete configuration was given to the filter has been resolved.
+       - An issue that could cause the fledge-filter-metadata to terminate a south service if very late integer values were used has been resolved.
+       - The fledge-filter-metadata plugin has been updated to support nested values to be defined.
+       - A problem that could result in excessive memory use when the fledge-filter-delta plugin is used.
+       - An issue with the fledge-north-opcuaclient plugin trying to write data to OPC UA nodes that didn't exist in the server has been updated.
+       - An issue with the statistics created by the fledge-north-opcuaclient plugin has been resolved. Previously the statistics may increase when data is not being sent.  
```

Comment:
fledge-north-opcuaclient:
- An enhancement has been implemented to resolve an issue where the plugin was trying to write data to OPC UA nodes that do not exist on the server.
- An issue concerning the statistics produced by the plugin has been rectified. Previously, these statistics could increase even in the absence of data transmission.